apm-sniffer/apm-sdk-plugin/dbcp-2.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/dbcp/v2/PoolingCloseConnectInterceptor.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.apache.skywalking.apm.plugin.dbcp.v2;
+
+import org.apache.skywalking.apm.agent.core.context.ContextManager;
+import org.apache.skywalking.apm.agent.core.context.trace.AbstractSpan;
+import org.apache.skywalking.apm.agent.core.context.trace.SpanLayer;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceMethodsAroundInterceptor;
+import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.MethodInterceptResult;
+import org.apache.skywalking.apm.network.trace.component.ComponentsDefine;
+
+import java.lang.reflect.Method;
+
+public class PoolingCloseConnectInterceptor implements InstanceMethodsAroundInterceptor {
+
+    @Override
+    public void beforeMethod(EnhancedInstance objInst, Method method, Object[] allArguments, Class<?>[] argumentsTypes, MethodInterceptResult result) throws Throwable {
+        AbstractSpan span = ContextManager.createLocalSpan("DBCP/Connection/" + method.getName());
+        span.setComponent(ComponentsDefine.DBCP);
+        SpanLayer.asDB(span);
```

Comment:
Connection pool is not a DB layer.

Reply:
Dear Mr Wu @wu-sheng  ,sorry to bother you, but could you give me some help?

Reply:
> Dear Mr Wu @wu-sheng ,sorry to bother you, but could you give me some help?

He is on vocation recently.

From the functionality of connection pool, I think it's ok to consider it as cache layer.

Reply:
> > Dear Mr Wu @wu-sheng ,sorry to bother you, but could you give me some help?
> 
> He is on vocation recently.
> 
> From the functionality of connection pool, I think it's ok to consider it as cache layer.
I got it！thanks for your reply！

Reply:
We don't.require layer for all plugins. For the connection pool, layer is not required.

Reply:
> We don't.require layer for all plugins. For the connection pool, layer is not required.

Thank you sir, and wish you a happy vacation!